local/notebooks/analysis.py

```diff
@@ -11,21 +11,35 @@
 def run_state_dict(datasets, label, mask, save_fig):
     """Runs an analysis across state for multiple dictionaries
     
-    datasets: list of dict of 4d arrays
+    datasets: list of dict
```

Comment:
I would write this as:
`list of dict, as [trial_data_dict, rest_data_dict]`
`each dict entry is a band with 3d array, as [subjects, channels, features]`